src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -285,9 +291,9 @@ public virtual VerifyResult VerifyForEachBlock(StoreView snapshot, BigInteger to
             return VerifyResult.Succeed;
         }
 
-        public virtual VerifyResult Verify(StoreView snapshot, BigInteger totalSenderFeeFromPool)
+        public virtual VerifyResult VerifyStateIndependent(StoreView snapshot)
```

Comment:
We shouldn't remove `Verify` method, which will contains `VerifyStateIndependent` and `VerifyStatedependent`.

Reply:
Have modified correspondingly, plz have a check:)